bftclient/test/bft_client_api_tests.cpp

```diff
@@ -301,15 +302,16 @@ TEST_F(ClientApiTestFixture, test_ignore_reply_from_wrong_replica) {
   auto WrongReplicaBehavior = [&](const MsgFromClient& msg, IReceiver* client_receiver) {
     auto reply = replyFromRequest(msg);
     if (!sent_reply_from_wrong_replica) {
-      client_receiver->onNewMessage(bad_replica_id, (const char*)reply.data(), reply.size());
+      client_receiver->onNewMessage(bad_replica_id, (const char*)reply.data(), reply.size(), bad_replica_id);
       sent_reply_from_wrong_replica = true;
     } else {
       // We have to update the bool first, since this runs in a separate thread from the test below. Otherwise we could
       // check the value of the bool after the match, but before the bool was set. Setting it early does no harm, as the
       // intention is just to show that the callback fired, which it clearly will. The code will block until the
       // callback fires anyway. Comment out the callback and rerun if you don't believe me :)
       sent_reply_from_correct_replica = true;
-      client_receiver->onNewMessage((NodeNum)msg.destination.val, (const char*)reply.data(), reply.size());
+      client_receiver->onNewMessage(
+          (NodeNum)msg.destination.val, (const char*)reply.data(), reply.size(), (NodeNum)msg.destination.val);
```

Comment:
No need to pass the same value twice - let the default value take place.

Reply:
updated.